x-pack/filebeat/input/awss3/input.go

```diff
@@ -376,5 +380,16 @@ func getProviderFromDomain(endpoint string, ProviderOverride string) string {
 	return "unknown"
 }
 
+func PollSqsWaitingMetric(ctx context.Context, receiver *sqsReader) {
```

Comment:
Does this need to be exported?

Reply:
I'm not really sure, but I don't think so?

Reply:
Let's make it unexported.